### ISSUE TITLE
Add automated input for TAS/testing

### ISSUE THREAD

**Issue by JMcKiern** [JMcKiern/NES-Emulator#34]

**Comment by INDOXABLE** [comment]
Lol.


I mean, you can make a script yourself for automating input events...

**Comment by JMcKiern** [comment]
Yup, the idea is to have a file with a bunch of controller actions that the emulator performs and then get the screen output to see if it matches what's expected.

It would allow for much more extensive tests
